app/models/groups/mailman/sync_listener.rb

```diff
@@ -112,13 +112,15 @@ def groups_affiliation_committed(affiliation)
       private
 
       def sync_memberships_for_groups_in_same_communities_if_admin_or_mod_perms(group)
+        return unless group.present?
```

Comment:
Another place where an exception was getting raised during cluster.destroy due to a nil group